src/Visualizer/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "vscode",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^17.0.2",
```

Comment:
IANAL, but let me write a note.

A few years ago, react's license was not good for us because of patent. But it seems to be changed to `MIT`. See https://www.asapdevelopers.com/react-native-update-to-mit-license/ for more informaiton, I am not sure whether other dependencies are safe to use.

Reply:
React is MIT license and has become a widely used framework. Can I ask you what kind of problems you're worried about?

Reply:
I checked that all the modules in `dependencies` have MIT or Apache 2.0 License.
But not for package-lock.json.
I will check it.

Reply:
> React is MIT license and has become a widely used framework. 

I already am aware of. Please find my comment I've already written. : )

> Can I ask you what kind of problems you're worried about?

Please find what I've written —  _"I am not sure whether other dependencies are safe to use."_.
@llFreetimell answered for my question. He checked other modules' licenses.